lms/templates/wiki/includes/editor_widget.html

```diff
@@ -1,6 +1,6 @@
 {% load i18n %}
 <textarea {{ attrs }}>{{ content }}</textarea>
-<p id="hint_id_content" class="help-block">
+<p id="hint_id_content" aria-describedby="hint_id_content" class="help-block">
```

Comment:
I thought we were putting `aria-describedby` on the textarea to link it to this `<p>`?

Reply:
Couldn't remember the deal. Will update.